README.md

```diff
@@ -62,6 +62,19 @@ The general requirements for new Component development:
 
 **NOTE: To manually publish a new version to the NPM, you will need the credentials that can be found from the City of Helsinki Culture and Leisure's Vault-service.**
 
+
+### Development environments
+
+You can use docker local environment for development:
+
+`docker compose up --build` when using Docker
+
+`podman compose up --build` when using Podman
```

Comment:
I think that every podman user should know this without explicitly mentioning about it. Docker is what people knows.